### ISSUE TITLE
Wrong composer.json config directory for symfony 3.x

### ISSUE THREAD

**Issue by HeahDude** [symfony/symfony-installer#277]
It seems the directory structure is wrong for symfony > 3.0.

ref https://github.com/sensiolabs/SensioDistributionBundle/issues/283.

**Comment by javiereguiluz** [comment]
I can't reproduce this error. If I execute `symfony new my_project`, the `extra.symfony-var-dir` option is `var` and not `app`

**Comment by jzawadzki** [comment]
@HeahDude - Just tried it and also have `extra.symfony-var-dir` correctly set to `var/` 
but I went through steps you posted  in original issue and I found why you had your `bootstrap.php.cache` created in `app/` directory.

Even if your `composer.json` `symfony-var-dir` would be correct, issue still would happen because you have removed `var/` directory.
When `symfony-var-dir` directory is not found, then `buildBootstrap` script is using `symfony-app-dir` directory instead. 
See line: https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Composer/ScriptHandler.php#L448
